### ISSUE TITLE
wrong worksheet is read if chart objects are present

### ISSUE THREAD

**Issue by t-kalinowski** [hadley/readxl#200]
Not sure how to make this reproducible, I can DM the excel sheet if desired but don't feel comfortable posting it publicly. 

This seems related to skipping worksheets that are chart objects.

```
path <- "2016-07-27_Site Wide Transducer Data_ID.xlsm"
sheet_names <- readxl::excel_sheets(path)
sheet_names[1:5]
# [1] "Well IDs"      "Barotroll-C"   "Barotroll"     "MW-Morris1A-C" "MW-Morris1A"  
readxl::read_excel(path = path, sheet = "Barotroll")) # i.e., sheetnames[3]
# should read in "Barotroll" but instead reads in "MW-Morris1A", i.e., sheetnames[5]
# sheets 2 and 4 are chart objects
```

**Comment by jennybc** [comment]
Likely related to #80 #104 #168

**Comment by jennybc** [comment]
Duplicate of #116, which provides a reprex.

**Comment by jennybc** [comment]
Will be handled as fix of #104
